spec/services/continue_with_facebook_service_spec.rb

```
@@ -3,42 +3,46 @@
-
-  describe '#verify_signature' do
-    let(:facebook_app_secret) { 'secret' }
-    let(:digested_encoded_payload) { 'digest' }
+  describe '#facebook_user_info' do
+    let(:facebook_user_info) { { 'id' => 123 } }
+    let(:decoded_payload) { { 'user_id' => 123 } }
+    let(:response) { double('response', body: facebook_user_info.to_json) }
 
     before do
-      ENV['PHOTONIA_FACEBOOK_APP_SECRET'] = facebook_app_secret
+      allow(instance).to receive(:valid_signature?).and_return(true)
```

rubocop (1.69.1)

[convention] Use `receive_messages` instead of multiple stubs on lines [22].

(RSpec/ReceiveMessages)